### DISCUSSION TITLE
Create a new package : Beeper

### DISCUSSION THREAD

**Discussion by labrute** [bcurran3/ChocolateyPackages_CCR#335] [Ideas]
I am reaching out to you with a request regarding the creation of a Chocolatey package for the Beeper application.

As an avid user and enthusiast of Beeper, I believe it would greatly benefit the community to have a Chocolatey package readily available. 

The Beeper application has gained significant popularity due to its versatile messaging capabilities, allowing users to consolidate their communication channels into a single interface. 

If you have the time and resources available, I kindly request your assistance in creating a Chocolatey package for Beeper. Your expertise in package creation would be immensely valuable in ensuring a seamless installation experience for the community.

Should you require any specific information or assistance from my end, please do not hesitate to reach out. I am more than willing to provide any necessary details or collaborate to facilitate this process.

Software can be downloaded here : [Beeper](https://www.beeper.com/download)

Thank you for considering my request. I greatly appreciate your time and effort.

Best regards,

**Comment by bcurran3** [comment]
Bonjour (encore) JF,

Never heard of it. I'll take it into consideration. What I'd probably do is create it as a "starter package" that I setup and then pass off to someone else to continue maintenance. This could be you! Since you're enthusiastic about it, you'd be the perfect candidate. I will guide you if you're interested.

**Reply by labrute** [reply]
Hi, 
I'm interested by helping you to develop this package. 
If you have a starter kit (things to install, how to test the package ...) it will be be usefull.

Thx.

**Comment by bcurran3** [comment]
Closing this as I am no longer creating or maintaining Chocolatey packages. Sorry, out
of my control.

Hopefully you've enjoyed my work and found it useful.

[![Bella Ciao!](https://img.youtube.com/vi/9ao4FEaDGhQ/0.jpg)](https://www.youtube.com/watch?v=9ao4FEaDGhQ)

I'm sure you know this but Chocolatey has a Google Groups where you can request someone to package up and maintain a program. If this isn't already in the CCR, you should request it there.
